ui/app/pages/create-account/external-account/index.js

```diff
@@ -0,0 +1,38 @@
+const inherits = require('util').inherits
+const Component = require('react').Component
+const h = require('react-hyperscript')
+const PropTypes = require('prop-types')
+const connect = require('react-redux').connect
+
+// Subviews
+const ExternalAccountImportView = require('./external-account.js')
```

Comment:
anything called something like "external sign(er)" should be renamed something like "BidirectionalQrSign(er)"